editor/actions.js

```diff
@@ -96,8 +95,11 @@ export function savePost( dispatch, postId, edits ) {
 
 export function trashPost( dispatch, postId, postType ) {
 	new wp.api.models.Post( { id: postId } ).destroy().done( () => {
-		window.location.href = 'edit.php?post_type=' + postType
-			+ '&trashed=1&ids=' + postId;
+		window.location.href = getWPAdminURL( 'edit.php', {
```

Comment:
Could this just reuse `addQueryArgs`, e.g. `addQueryArgs( 'edit.php', { post_type: postType } );` ?